src/schema/member.ts

```diff
@@ -4,22 +4,21 @@ import { Member as DefinedMember } from '@/database/types'
 
 export { zodResolver } from 'mantine-form-zod-resolver'
 
-export const schema = z.object({
+export const memberSchema = z.object({
     identifier: z.string().regex(/^[a-z][a-z\-]*[a-z]$/, { message: 'Invalid identifier, all lowercase, only dashes' }),
     name: z.string().min(1, { message: 'Name must be provided' }),
     email: z.string().email({ message: 'Invalid email address' }),
     publicKey: z.string().min(1, { message: 'PubKey cannot be blank' }),
 })
 
-export type ValidatedMember = z.infer<typeof schema>
+export type ValidatedMember = z.infer<typeof memberSchema>
 
 export const NEW_MEMBER: NewMember = {
     identifier: '',
     name: '',
     email: '',
     publicKey: '',
 }
-Object.freeze(NEW_MEMBER)
```

Comment:
This was causing an issue on the add new member form, this shouldn't be necessary 🤔

Reply:
I think this should stay as a safeguard.  If the NewMember object gets mutated then the next user of the form will see the previous users data 😱 

If the new member form complains, you can use `initialValues: { ...NEW_MEMBER }` to clone it

Reply:
I see what you're worried about, but having tested it without the freeze, filling out the form, submitting and cancelling the form, and then reopening the add member form it will be cleared out entirely. I don't know if there's any edge case im missing here in testing though. However I thought of maybe a simpler solution, we can just have a method that returns a "new member" with the blank object, then pass that into the component, rather than having the internal form spread the values of any member it gets
```
export const getNewMember = (): NewMember => {
    return {
        identifier: '',
        name: '',
        email: '',
        publicKey: '',
    }
}
```

let me know how that sounds!

Reply:
yah that sounds great!  basically this entire thing was an attempt to get a new blank member for forms which that accomplishes.  Really it's probably an over-optimization and the InitialValues could just be specified in the useForm vs imported from here